### DISCUSSION TITLE
How can I add information to the explorer listing by clicking the 'make changes' button?

### DISCUSSION THREAD

**Discussion by theo-ahnlabio** [WalletConnect/walletconnect-monorepo#2548] [Developer Support] [answered]
I have an explorer listing that I previously submitted. 
There is additional information I want to update on the listing, 
so I press "make change" to add it. However, an error message appears.

![screenshot_wcex_1](https://github.com/WalletConnect/walletconnect-monorepo/assets/107841222/42451e73-9263-4d17-b382-68117324812e)

error message--> Updating the listing failed: "new row violates row-level security policy for table \"registry submission! ""

<img width="587" alt="screenshot_wcexplorer" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/107841222/b44f3018-5baf-4343-83c7-36ca0d55f1a8">

How can I add information to the explorer listing by clicking the 'make changes' button?

**Comment by Cali93** [comment, marked as answer]
Hi @theo-ahnlabio,

Thanks for reporting this issue.
It has been fixed a few minutes ago and you should now be able to update your submission.

Sorry for the inconvenience.
